core/src/main/scala/Monitoring.scala

```diff
@@ -160,10 +183,13 @@ trait Monitoring {
 
           urlSignals.put(source, hook)
 
+          val removeFromMirroringUrls: Process[Task, Unit] =
```

Comment:
this is new code.  I can put it locally within processMirroringEvents assuming that's what you mean